tf_patches/ffp_gpu.diff

```diff
@@ -0,0 +1,40 @@
+# enable gpu build, remove after next pin update
+diff --git a/tensorflow/core/kernels/fft_ops.cc b/tensorflow/core/kernels/fft_ops.cc
+index 5e0464e89f1..2d9cd438eb2 100644
+--- a/tensorflow/core/kernels/fft_ops.cc
++++ b/tensorflow/core/kernels/fft_ops.cc
+@@ -455,7 +455,7 @@ class FftPlanCache {
+       cache_.erase(it);
+     }
+     --size_;
+-    return value;
++    return std::optional<Value>(std::move(value));
+   }
+ 
+   // Inserts a plan into the cache as long as there is still capacity.
+diff --git a/tensorflow/compiler/xla/runtime/ffi.cc b/tensorflow/compiler/xla/runtime/ffi.cc
+index 584d2008b11..231abca9c92 100644
+--- a/tensorflow/compiler/xla/runtime/ffi.cc
++++ b/tensorflow/compiler/xla/runtime/ffi.cc
+@@ -380,7 +380,7 @@ absl::StatusOr<FfiStateVector> FfiModulesState::state_vector() const {
+ 
+     return absl::InternalError("Unsupported FFI module state");
+   }
+-  return state_vector;
++  return std::move(state_vector);
```

Comment:
Nit: Could you add a comment about why `std::move` is needed for us in these cases?